pkg/apis/core/types.go

```diff
@@ -2951,6 +2951,16 @@ const (
 	// DisruptionTarget indicates the pod is about to be terminated due to a
 	// disruption (such as preemption, eviction API or garbage-collection).
 	DisruptionTarget PodConditionType = "DisruptionTarget"
+	// PodResizePending indicates that the pod has been resized, but kubelet has not
+	// yet allocated the resources. If both PodResizePending and PodResizeInProgress
+	// are set, it means that a new resize was requested in the middle of a previous
+	// pod resize that is still in progress.
+	PodResizePending PodConditionType = "PodResizePending"
+	// PodResizeInProgress indicates that a resize is in progress, and is present whenever
+	// allocated resources != acknowledged resources. If both PodResizePending and
```

Comment:
"acknowledged resources" is an internal implementation detail. Better to describe the state, something like:
```suggestion
	// the Kubelet has allocated resources for the resize, but has not yet actuated all of the required changes.
```

Reply:
done!